### PR TITLE
fix: add vpc outputs

### PR DESCRIPTION
**Pull request by ajp-lqx**
#### Description
fixes: #188

### CODE REVIEW

**Comment by ankitm123** [comment]
Need to re-generate readme file.
```
terraform-docs markdown table .
```

Also, would be awesome to just add a basic check to see if subnets were indeed created: https://github.com/jenkins-x/terraform-aws-eks-jx/blob/master/test/terraform_eks_test.go

We use the jx3 example repo for testing purposes: https://github.com/jenkins-x/terraform-aws-eks-jx/tree/master/examples/jx3 (you can update the `output.tf` with the vpc output)

File: modules/cluster/outputs.tf [hunk 1]
```diff
@@ -11,6 +11,14 @@ output "cluster_oidc_issuer_url" {
   value = local.oidc_provider_url
 }
 
+// ----------------------------------------------------------------------------
+// VPC Outputs
+// ----------------------------------------------------------------------------
+output "vpc" {
+  description = "All outputs from the VPC module"
+  value       = module.vpc
```
Comment (ankitm123) [inline modules/cluster/outputs.tf:19]: https://github.com/terraform-aws-modules/terraform-aws-vpc/blob/master/outputs.tf -> This is the list of output that you can use from. So if you just want the id, `module.vpc.id`

**Comment by ajpauwels** [comment]
We should probably just close this @ankitm123

**Pull request closed without merging**